packages/runtime/test/transport/relationships.test.ts

```diff
@@ -82,6 +82,20 @@ describe("Create Relationship", () => {
     });
 
     describe("tests on pending relationship", () => {
+        test("should not be able to create a new relationship if a pending relationship already exists", async () => {
```

Comment:
This test isn't successful on its own, as it is dependent on "create pending relationship". However, I noticed that this is the case for also other tests, so we should take care of this in a separate PR.

Reply:
You are absolutely right that this is not good. This should be adjusted in a separate PR.